packages/split-button/src/Menu/Menu.tsx

```diff
@@ -58,9 +58,7 @@ export const Menu = ({
 
   const handleTriggerClick: MouseEventHandler = e => {
     onTriggerClick?.(e);
-    if (typeof controlledOpen !== 'boolean') {
-      setOpen(o => !o);
-    }
+    setOpen(o => !o);
```

Comment:
@shaneeza do you remember the context for why this was conditional on `controlledOpen` when adding it in https://github.com/mongodb/leafygreen-ui/commit/586703fc4f3ed05b400b29ccf53c99a24ce6dd93?

Reply:
Hey, I am trying to remember why this was conditional. For clarity, because of this check, your controlled `setOpen` was not being called. Is that correct?

Taking a look at [Menu](https://github.com/mongodb/leafygreen-ui/blob/main/packages/menu/src/Menu/Menu.tsx#L248), we don't include this check, so there is a chance we probably don't need this check.

Reply:
> because of this check, your controlled setOpen was not being called. Is that correct?

Correct.